frameworks/Rust/may-minihttp/may-minihttp.dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM rust:1.65
+FROM rust:latest
```

Comment:
Can you please set this to a specific version. I think a few have gotten through with `latest` that need to be updated.

Reply:
changed to rust:1.66